### PR TITLE
Update CS translation.json

### PR DESCRIPTION
**Pull request by VasaXter**
Complete Czech translation

### CODE REVIEW

File: src/locales/cs/translation.json [hunk 1]
```diff
@@ -442,154 +442,154 @@
...
                 "subtitle": "$t(communityLinks.contentType.member)",
-                "desc": "3D printer files for card organizers to better fit in the Big Box, as well as token organizers.",
+                "desc": "Soubory pro 3D tisk pro organizéry na karty nebo tokeny pro lepší umístění v Big boxu.",
                 "link": "https://thingiverse.com/thing:6078120"
             }
         ]
     },
-    "lostMount": {
-        "title": "Lost Mount Locations"
+    "Mount": {
```
Comment (swirle13) [inline src/locales/cs/translation.json:540]: This needs to stay as "lostMount" to not break references to this title used elsewhere.

```suggestion
    "lostMount": {
```

**Review by swirle13 — CHANGES_REQUESTED** [review]
Very good! Just one small change and I'll merge it into the live website!

File: src/locales/cs/translation.json (hunk 1 above)
Comment (VasaXter) [reply src/locales/cs/translation.json:540]: I don't even know how this happened :-D, thanks for checking and fixing

File: src/locales/cs/translation.json [hunk 2]
```diff
@@ -1,82 +1,82 @@
...
+        "card_witchermagetrophy_many": "{{count}} karty trofeje zaklínače/mága",
+        "card_witchermagetrophy_other": "{{count}} karet trofeje zaklínače/mága",
+        "token_level1_one": "{{count}} žeton úrovně I",
+        "token_level1_few": "{{count}} žetony úrovně I",
+        "token_level1_many": "{{count}} žetonu úrovně I",
+        "token_level1_other": "{{count}} žetonů úrovně I",
+        "token_shield_one": "{{count}} žeton Štítu",
+        "token_shield_few": "{{count}} žetony Štítu",
+        "token_shield_many": "{{count}} žetonů Štítu",
+       "token_shield_other": "{{count}} žetonu Štítu",
```
Comment (swirle13) [inline src/locales/cs/translation.json:34]: Nit: this indent needs to be in line with the rest of the keys

```suggestion
        "token_shield_other": "{{count}} Shield Tokens",
```

File: src/locales/cs/translation.json [hunk 3]
```diff
@@ -334,106 +334,106 @@
...
-            "dagonMT": "Place the Dagon miniature on the starting space corresponding to player count. Shuffle the Dagon Bonus cards and place them, Dagon monster card (4 Special Attacks face-up), and Dagon Trail tokens near its track.",
-            "siren": "Place the Siren miniature on the Dagon location, with its Monster Card nearby."
+            "decks": "Zamíchejte balíček průzkumu Skellige. Nemíchejte balíček událostí Skellige! Oba umístěte na jejich místo, nad ostrovy na herním plánu.",
+            "dagon": "Umístěte figurku Dagona na počáteční místo dle počtu hráčů. Zamíchejte bonusové karty Dagona a umístěte je, kartu nestvůry Dagona (speciální schopností nahoru) a žetony stopování Dagona poblíž stupnice nebezpečí.",
+            "dagonMT": "Umístěte figurku Dagona na počáteční místo dle počtu hráčů. Zamíchejte bonusové karty Dagona a umístěte je, kartu nestvůry Dagona (čtyřmi speciálními útoky nahoru) a žetony stopování Dagona poblíž stupnice nebezpečí.",
+            "siren": "Umístěte figurku Sirény na Dagonovu lokaci a poblíž její kartu nestvůry.",
```
Comment (swirle13) [inline src/locales/cs/translation.json:374]: Trailing comma needs to be removed. It causes an issue where it expects another entry but finds none and won't successfully compile.

```suggestion
            "siren": "Umístěte figurku Sirény na Dagonovu lokaci a poblíž její kartu nestvůry."
```

**Review by swirle13 — CHANGES_REQUESTED** [review]
Two small punctuation issues, which I can update myself

**Review by swirle13 — APPROVED** [review]
Looks good!

**Pull request merged**